app/src/handlers/census/CensusProxy.ts

```diff
@@ -23,6 +28,11 @@ export default class CensusProxy {
     private readonly continentLockHandler: ContinentLockEventHandler;
     private readonly facilityControlEventHandler: FacilityControlEventHandler;
     private readonly gainExperienceEventHandler: GainExperienceEventHandler;
+    private readonly achievementEarnedHandler: AchievementEarnedHandler;
```

Comment:
These were missing? 😲

Reply:
Yep, not sure what happened. Will include some tests later to catch such errors